extension/src/paymentHandler/payment-utils.js

```diff
@@ -59,25 +59,25 @@ function getMatchingCtpState (adyenState) {
 
 function createAddInterfaceInteractionAction (
   {
-    request, response, type, status
+    request, response, type
   }
 ) {
   // strip away sensitive data
   delete response.additionalData
 
   return {
     action: 'addInterfaceInteraction',
-    type: { key: c.CTP_INTERFACE_INTERACTION },
+    type: { key: c.CTP_INTERFACE_INTERACTION_PAYMENT_TYPE_KEY },
     fields: {
-      createdAt: new Date(),
+      createdAt: new Date().toISOString(),
       response: JSON.stringify(response),
-      request: JSON.stringify(request.body),
-      type,
-      status
+      request: request.body,
```

Comment:
You do not need to escape the body JSON or it will be escaped automatically?

Reply:
you don't but the same concern as here: https://github.com/commercetools/commercetools-adyen-integration/pull/232#discussion_r405636607. I will try to change those to find a better structure.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/232/commits/6777ff34a8005fa3b01970789ff2146018efc635